pyproject.toml

```diff
@@ -7,13 +7,13 @@ license = "MIT"
 packages = [{ include = "tellor_disputables", from = "src" }]
 
 [tool.poetry.dependencies]
-python = "^3.9.2"
+python = ">=3.9,<3.11"
```

Comment:
is there support for 3.10 now?

Reply:
not exactly sure, but this was a recommendation on the cli that worked when added. Some of the dependencies were failing during install when telliot-feeds version was bumped